src/mmrelay/db_utils.py

```
@@ -682,28 +708,83 @@ def _store(cursor: sqlite3.Cursor) -> None:
 
     try:
         manager.run_sync(_store, write=True)
+        return True
```

Ruff (0.15.4)

[warning] Consider moving this statement to an `else` block

(TRY300)